assets/src/edit-story/components/checklist/tips/pageTooMuchText.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+// import { __ } from '@web-stories-wp/i18n';
+import { useMemo } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../../app/story';
+import { characterCountForPage, filterStoryPages } from '../utils';
+// import ChecklistCard from '../../../checklistCard';
+
+const MAX_PAGE_CHARACTER_COUNT = 200;
+
+/**
+ * @typedef {import('../../../types').Page} Page
+ */
+
+/**
+ * Check page for too much text
+ *
+ * @param {Page} page The page being checked for guidelines
+ * @return {boolean} returns true if page has more text than MAX_PAGE_CHARACTER_COUNT
+ */
+export function pageTooMuchText(page) {
+  return characterCountForPage(page) > MAX_PAGE_CHARACTER_COUNT;
+}
+
+export function StoryPosterAttached() {
+  const story = useStory(({ state }) => state);
+  const failingPages = useMemo(
+    () => filterStoryPages(story, pageTooMuchText),
+    [story]
+  );
+  return failingPages.length > 0
+    ? // <ChecklistCard
+      //   title={__('Reduce amount of text on pages', 'web-stories')}
```

Comment:
What are your thoughts on using the `MESSAGES` constant for the text still? I feel like it might be helpful to continue doing that for the sake of tests. Happy to have that happen later not here.  Just wanted to bring it up.

Reply:
I think the `MESSAGES` constant gets a bit bloated and hard to navigate once you get above like 10 tips. I don't this new structure will have any impact on the unit tests for the checks because the new functions will now just return `true || false` instead of `Guidance || undefined`

Reply:
Yeah i was originally thinking we should keep the constants, too. But I was thinking back on some tests that failed because we were looking for a string that no longer matched when the copy changed a lot the first time though and now think.... that's probably good that they failed because we weren't matching on a constant. it helps highlight what changed (when the copy inevitably changes) and asks us to verify our changes more explicitly.

Reply:
just to clarify, is the ask here to use the `MESSAGES` constant or to have the filter functions return `Guidance` or `undefined` like they used to? or both?

Reply:
i was just talking about the messages - purely the strings from the spreadsheet for issue copy.

Reply:
oh I remember this. forgot about that. thanks for reminding me on the context.

Is the spread sheet still the single source of truth for copy? or are the figma files?

Reply:
but yea totally makes sense to keep all the copy in a central constant if we're scanning it against a spreadsheet

Reply:
spreadsheet!